### ISSUE TITLE
Clicking on Dblookup Mediator in Design View results in UI Crash

### ISSUE THREAD

**Issue by viggnah** [wso2/mi-vscode#331]
**Error**
Getting the below error when I click on the Dblookup mediator - 
![Screenshot 2024-08-08 at 1 50 37 PM](https://github.com/user-attachments/assets/9115cbfd-f551-4e12-af4f-8cc827618088)

**Steps to reproduce:**
1. Using the following code generated by MI Copilot - 
```xml
<api xmlns="http://ws.apache.org/ns/synapse" name="DynamicEndpointRouter" context="/dynamicRouter">
    <resource methods="GET" uri-template="/route?serviceId={serviceId}">
        <inSequence>
            <property name="serviceId" expression="$url:serviceId" scope="default" type="STRING"/>
            <dblookup>
                <connection>
                    <pool>
                        <driver>com.mysql.jdbc.Driver</driver>
                        <url>jdbc:mysql://localhost:3306/endpointDB</url>
                        <user>user</user>
                        <password>password</password>
                    </pool>
                </connection>
                <statement>
                    <sql>SELECT endpointUrl FROM Endpoints WHERE serviceId = ?</sql>
                    <parameter expression="get-property('serviceId')" type="VARCHAR"/>
                    <result name="endpointUrl" column="endpointUrl"/>
                </statement>
            </dblookup>
            <property name="dynamicEndpoint" expression="get-property('endpointUrl')" scope="default" type="STRING"/>
            <send>
                <endpoint>
                    <address uri="{get-property('dynamicEndpoint')}"/>
                </endpoint>
            </send>
        </inSequence>
    </resource>
</api>
```

**Affected Versions:**
1.0.3

**OS, DB, other environment details and versions:**
MacOS 14.4.1 (23E224)

**Comment by thuva9872** [comment]
The latest release v1.0.5 has resolved the issue. Please reopen if the problem persists.

**Comment by viggnah** [comment]
Sorry to re-open but is v1.0.5 released yet? I can only see v1.0.4 still on VS Code..
![Screenshot 2024-09-05 at 10 41 19 AM](https://github.com/user-attachments/assets/818cd264-36e6-436e-a5c3-125956da5dad)

**Comment by thuva9872** [comment]
Hi @viggnah ,

Due to technical issues in the VS Code Marketplace, the latest version (v1.0.5) has not yet been released there. However, you can download it from the Open VSX Registry [here](https://open-vsx.org/extension/wso2/micro-integrator). The upcoming versions will be released in the VS Code marketplace itself. Sorry for the inconvenience.

Regards,
Thuvarakan
